install_config/configuring_authentication.adoc

```diff
@@ -200,6 +201,59 @@ identity providers are configured that identify the same set of users and map to
 the same user names.
 |===
 
+[[LookupMappingMethod]]
```

Comment:
Normally we have been using lowercase for anchor tags with '-'. Any particular reason this could not have been lookup-mapping-method?

Reply:
@vikram-redhat   I do not recall. But, I likely modelled it after the anchor tags that followed in the file, like: [[AllowAllPasswordIdentityProvider]] and [[DenyAllPasswordIdentityProvider]]